packages/frontend-2/pages/index.vue

```diff
@@ -1,20 +1,159 @@
 <template>
-  <ProjectsDashboard v-if="isLoggedIn" />
-  <div v-else class="mx-auto">
-    <CommonLoadingIcon />
+  <div>
+    <Portal to="navigation">
+      <HeaderNavLink :to="homeRoute" name="Dashboard" hide-chevron :separator="false" />
+    </Portal>
+    <PromoBannersWrapper v-if="promoBanners.length" :banners="promoBanners" />
```

Comment:
There's some conflicts w/ dashboard.vue, it has a different header now in `main`